benchmarks/README.md

```diff
@@ -10,30 +10,53 @@ via these terraform scripts on a Standard cluster that you've created yourself.
 
 ## Pre-requisites
 
+### gcloud auth
+
+This tutorial assumes you have access to use google storage APIs via Application Default Credentials (ADC).
+To login, you can run the following:
+
+```
+gcloud auth application-default login
+```
+
 ### Terraform
 
 Install Terraform by following the documentation at https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli.
 This requires a minimum Terraform version of 1.7.4
 
-## Deploy and serve an ML model
+### python
+
+Python script used in this tutorial assumes in your environment you are running Python >= 3.9
+
+You may need to run pip install -r requirements.txt to install the library dependencies. (Optionally, you can run this within a venv, i.e. python3 -m venv ./venv && source ./venv/bin/activate && pip install ...)
```

Comment:
what's the dir path towards`requirements.txt`?

Reply:
applied Ashok's suggestion :) ty both for the review!